src/main/java/uk/gov/hmcts/juror/api/juror/domain/JurorResponseQueries.java

```diff
@@ -97,7 +97,7 @@ public static BooleanExpression byAssignmentAndProcessingStatusAndUrgency(String
      */
 
     private static BooleanExpression backlogUrgent() {
-        return jurorResponse.urgent.isTrue().and(jurorResponse.superUrgent.isFalse());
+        return jurorResponse.urgent.isTrue().and(jurorResponse.superUrgent.isNull());
```

Comment:
It will be worth having this as isNull or false (To account for both scenarios)

Reply:
added